controllers/disruption_cron_controller.go

```diff
@@ -10,6 +10,8 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/DataDog/chaos-controller/o11y/metrics"
```

Comment:
nit: I think you'll want to swap L14 and L15 here, so that all the "local" imports are in the middle group. we ought to figure out/fix our auto-importing